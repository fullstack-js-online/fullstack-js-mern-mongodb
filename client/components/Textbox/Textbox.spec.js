import React from 'react'
import { mount } from 'enzyme'

// components
import Textbox from './index'

describe('The Textbox component', () => {
  const props = {
    onChange: jest.fn(),
    placeholder: 'Email',
    value: 'john@doe.com'
  }

  it('should match snapshot', () => {
    expect(mount(<Textbox {...props} />)).toMatchSnapshot()
  })
})
