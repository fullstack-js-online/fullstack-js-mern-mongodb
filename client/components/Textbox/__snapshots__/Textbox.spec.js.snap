// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Textbox component should match snapshot 1`] = `
<Textbox
  onChange={[MockFunction]}
  placeholder="Email"
  value="john@doe.com"
>
  <div
    className="mb-5"
  >
    <input
      className="focus:outline-none w-full p-6 bg-brown-lightest text-brown"
      onChange={[MockFunction]}
      placeholder="Email"
      value="john@doe.com"
    />
  </div>
</Textbox>
`;
